stdlib/public/core/UnboundCapacity.swift

```diff
@@ -82,8 +82,19 @@ extension UnboundCapacity {
 
   /// Converts our contents into an array with at least the given
   /// capacity, and presents it to `body` for mutation.
-  internal mutating func withMutableArray<R>(
-    minCapacity: Int = 0,
+  //
+  // Michael NOTE: temporarily public, should probably make private
+  //
+  // Michael NOTE: forwarding version added due to odd linker behavior. Linker
+  // couldn't find the version with a default argument.
+  //
+  public mutating func withMutableArray<R>(
+    body: (inout [Base.Iterator.Element]) throws->R
+  ) rethrows -> R {
+    return try withMutableArray(minCapacity: 0, body: body)
+  }
```

Comment:
I don't really understand why you'd want these.  The only plausible reason would be that you plan to resize the array inside the closure, but I don't see you doing that anywhere.  Don't we really want `withUnsafeMutableBufferPointer`?